### PR TITLE
Enable apiary metastore metrics

### PR DESCRIPTION
**Pull request by spuranda123**

### CODE REVIEW

**Comment by rpoluri** [comment]
looks good, can you also update changelog

File: Dockerfile [hunk 1]
```diff
@@ -57,6 +57,7 @@ RUN cd /tmp && \
     tar xfz apache-atlas-${ATLAS_VERSION}-sources.tar.gz && \
     cd apache-atlas-sources-${ATLAS_VERSION}/ && \
     patch  -p1 < /tmp/atlas-${ATLAS_VERSION}-hive-2.3.3.patch && \
+    sed -s 's#http://repo1.maven.org#https://repo1.maven.org#' -i pom.xml && \
```
Comment (barnharts4) [inline Dockerfile]: Can you both a) update changelog with your changes, and b) put in the `changed` section why you had to make the above change?

File: files/startup.sh [hunk 2]
```diff
@@ -51,6 +51,8 @@ fi
 
 if [ ! -z $ENABLE_METRICS ]; then
     export ECS_TASK_ID=$(wget -q -O - http://169.254.170.2/v2/metadata|jq -r .TaskARN|awk -F/ '{ print $NF }')
+    # Set ECS_TASK_ID to K8 Hostname for EKS
+    export ECS_TASK_ID=${ECS_TASK_ID:-${HOST_NAME}}
```
Comment (barnharts4) [inline files/startup.sh]: I take it you decided to use hostname as the EKS ID for metrics?

File: Dockerfile (hunk 1 above)
Comment (spuranda123) [reply Dockerfile]: I will

File: files/startup.sh (hunk 2 above)
Comment (spuranda123) [reply files/startup.sh]: Yup, do you want me to make it otherwise ?

Comment (barnharts4) [reply files/startup.sh]: No, it's fine.  Just not super clear what is going on without some context.  Maybe add a line about that in the changelog too.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,14 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.6.0] -
+
+### Added
+- Set EKS hostname to ECS_TASK_ID required for enabling metatsore metrics.
```
Comment (barnharts4) [inline CHANGELOG.md]: `metatsore` -> `metastore`

**Pull request merged**